src/test/java/com/skynest/BaseTest.java

```diff
@@ -7,12 +7,13 @@
 import java.net.URISyntaxException;
 
 public class BaseTest {
+    public static final String TARGET_DOMAIN = "targetDomain";
```

Comment:
if you don't foresee this value being used outside of the setupEnvironment() you might as well make it a local variable. if you still decide to keep it as a constant, make it private.